docs/examples/frameworks/mxnet/mxnet-various-readers.ipynb

```diff
@@ -96,31 +96,26 @@
    "metadata": {},
```

Comment:
<p>I guess you didn't rerun the example? (the name of the pipeline has changed)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>